crypto-ffi/src/generic.rs

```diff
@@ -351,26 +373,26 @@ impl CoreCrypto<'_> {
     /// See [core_crypto::MlsCentral::mls_init]
     pub fn mls_init(&self, client_id: &ClientId) -> CryptoResult<()> {
         let ciphersuites = vec![MlsCiphersuite::default()];
-        future::block_on(self.executor.lock().map_err(|_| CryptoError::LockPoisonError)?.run(
-            self.central.lock().map_err(|_| CryptoError::LockPoisonError)?.mls_init(
-                client_id.clone(),
-                ciphersuites,
-                None,
+        future::block_on(
+            self.executor.lock().map_err(|_| CryptoError::LockPoisonError)?.run(
+                self.central
+                    .lock()
+                    .map_err(|_| CryptoError::LockPoisonError)?
+                    // weird but avoids importing 'either' crate here.
+                    .mls_init(Err(client_id.clone()).into(), ciphersuites),
```

Comment:
You could import `either` safely here; It's already in `core-crypto`'s dependency graph so it would just resolve to the same compilation unit.

Reply:
Yes, I wanted to avoid duplicating dependencies declaration but you're right, the workaround is too ugly